GearSpawner/GearSpawnReader.cs

```diff
@@ -24,19 +24,8 @@ internal static class GearSpawnReader
 		@"(?:\W+r\s*=\s*(" + VECTOR + "))?" +
 		@"(?:\W+\s*c\s*=\s*(" + NUMBER + "))?$");
 
-	private static string GetTrimmedLine(string line)
-	{
-		if (line == null)
-		{
-			return "";
-		}
-		else
-		{
-			// had to remove "ToLowerInvariant" as Asset refernces are case sensitive
-			return line.Trim();
-//			return line.Trim().ToLowerInvariant();
-		}
-	}
+
+	private static string GetTrimmedLine(string line) => line?.Trim() ?? "";
```

Comment:
If line is never null, this method can be simplified to `line.Trim()`, inlined, and removed.